doc/JSON.md

```diff
@@ -0,0 +1,18 @@
+STEPS TO BUILD AND COMPILE THE PROJECT
+=======================================
+
+After Installing the prerequisites
+
+Step 1 : We first need to build the sail parser.
```

Comment:
In markdown, you can get numbered lists with just putting literally `1.` before each item:
1. one (text starts with `1.`)
1. two (text starts with `1.`)

Use this method for both the main list of steps and the sub-lists.

Reply:
This is probably not a big deal, since GitHub seems to autoformat the _code_ (not just the rendering) with sequential numbers (1, 2, 3, ...) instead of (1, 1, 1, ...), but if you intended adopt the latter semantics, I don't see it that way.

For example, I expected the markdown for the steps to now look like:
```
1.  We first need to build the sail parser.
￼    1. Find a directory to clone this repository, and make that directory your current working directory.
￼    1. `git clone -b json https://github.com/ThinkOpenly/sail`
￼    1. `cd sail`
￼    1. `make`
1.  Then, set up the environment for building in the RISC-V Sail repository:
￼    1. `eval $(opam env)`
￼    1. `export PATH=<path-to-sail-repository>:$PATH`
[...]
```